api.py

```diff
@@ -22,9 +22,7 @@ async def status():
 
 logger = get_logger()
 
-AJV_HOST = os.getenv("AJV_HOST", "localhost")
-
-AJV_VALIDATOR_URL = f"http://{AJV_HOST}:5002/validate"
+AJV_VALIDATOR_URL = os.getenv("AJV_VALIDATOR_URL", "http://localhost:5002/validate")
```

Comment:
I think there is potential now to create a url that will break validator, in that you need to make sure the Port is 5002 since this is hardcoded in the app. So we probably need to handle this in some way.

Reply:
There won't be a port configuration when deployed in GCP as the backend service is referenced directly.

Reply:
But guess we still need to handle the local use case? For our dev and PR wokflows etc

Reply:
I have currently changed it set to: 
`AJV_VALIDATOR_URL = f"{AJV_VALIDATOR_SCHEME}://{AJV_VALIDATOR_HOST}:5002/validate"`